scripts/packages/e2e-test-create-block-app.ts

```diff
@@ -74,17 +75,33 @@ const script = async () => {
         blockName,
         "--path",
         blockDirPath,
+        ...(blockTemplate ? [`--template=${blockTemplate}`] : []),
       ],
       {
         ...defaultExecaOptions,
+        reject: false,
         cwd: os.tmpdir(),
       },
     );
 
+    if (
+      !blockTemplate ||
+      ["custom-element", "html", "react"].includes(blockTemplate)
+    ) {
+      if (result.exitCode) {
+        throw new Error(`Unexpected exit code ${result.exitCode}`);
+      }
+    } else {
+      if (!result.exitCode) {
+        throw new Error(`Expected non-zero exit code`);
+      }
+      return;
+    }
+
     logStepEnd();
     logStepStart("Check folder structure");
 
-    for (const relativePath of [".npmignore", "dist"]) {
+    for (const relativePath of [".npmignore", "dist", "dev"]) {
```

Comment:
Needs updating when `block-template-html/dev/index.html` is restored

Reply:
I see. Switched to `dev/src` in https://github.com/blockprotocol/blockprotocol/pull/480/commits/834f5c0a93bd2081a38781e6e4977649948409a5